cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1136,8 +1124,7 @@ public void updateSchedule(FullHttpRequest request, HttpResponder responder,
                              @PathParam("namespace-id") String namespaceId,
                              @PathParam("app-name") String appName,
                              @PathParam("schedule-name") String scheduleName) throws Exception {
-    doUpdateSchedule(request, responder, namespaceId, appName,
-                     getLatestAppVersion(new NamespaceId(namespaceId), appName), scheduleName);
+    doUpdateSchedule(request, responder, namespaceId, appName, ApplicationId.DEFAULT_VERSION, scheduleName);
```

Comment:
We're doing this because schedule is `versionless`?

Reply:
yes

Reply:
Are the changes to make schedules versionless already done? If so, it's better to just remove app version as an argument to doUpdateSchedule().

If not, this change is ok for now, with the expectation that the schedules PR will remove app version everywhere that it's not actually being used.

Reply:
Will leave this as is in this PR, and remove the version argument in the schedule PR